runtime/deferral_test.go

```diff
@@ -1192,7 +1274,24 @@ func TestRuntimeStorageDeferredResourceDictionaryValues_Insertion(t *testing.T)
 		getSigningAccounts: func() []Address {
 			return []Address{signer}
 		},
-		updateAccountCode: func(address Address, code []byte) (err error) {
+		resolveLocation: func(identifiers []Identifier, location Location) []ResolvedLocation {
```

Comment:
This function could be factored out, it's repeated quite a few times in this file. 
```suggestion
		resolveLocation: ContractAddressLocationResolver(t),
```

```go
func ContractAddressLocationResolver(t *testing.T)  func([]Identifier, Location) []ResolvedLocation {
  return func(identifiers []Identifier, location Location) []ResolvedLocation {
			require.Len(t, identifiers, 1)
			require.IsType(t, AddressLocation{}, location)

			return []ResolvedLocation{
				{
					Location: AddressContractLocation{
						AddressLocation: location.(AddressLocation),
						Name:            identifiers[0].Identifier,
					},
					Identifiers: identifiers,
				},
			}
		}
}
```

Reply:
Refactored in ad1298df5a98ff3760a9c1b3ebab14fe7a697646